sim/src/mechanics/intersection.rs

```diff
@@ -1084,6 +1084,7 @@ fn allow_block_the_box(i: &Intersection) -> bool {
         || id == 1726088130
         || id == 53217946
         || id == 53223864
+        || id == 53211694
```

Comment:
Are these OSM Id's? My understanding is those aren't technically stable identifiers, but they don't change that much in practice. Right?

Reply:
Correct on both. We also rely on OSM ID stability for map edits to last over time. Every now and then somebody splits  way near a traffic signal that we bake in, and I have to go manually redo it.

As far as I'm aware, there are no alternatives. :(  https://wiki.openstreetmap.org/wiki/Permanent_ID is just one proposal that never seemed to take off.

Reply:
Also this function would ideally be smarter and detect all cases when it's better to disable blocking-the-box at an intersection. We've got one case working -- degenerate intersections -- but there's more we could capture by looking for cases where blocking an intersection couldn't possibly bother the other direction, like with:
![Screenshot from 2021-04-06 16-09-46](https://user-images.githubusercontent.com/1664407/113788771-93bfba80-96f2-11eb-8181-4e458e612884.png)

Reply:
Unless those are real cross-walks - wouldn't want to encourage blocking them. =)

Reply:
Most of them aren't, but some are. #485 is still somewhere on the TODO, and eventually using the OSM data when it exists